psychopy/event.py

```diff
@@ -798,3 +807,83 @@ def clearEvents(eventType=None):
                             locals.JOYBUTTONDOWN])
         else:
             junk = evt.get()
+
+
+_GlobalKeyEvent = namedtuple('_GlobalKeyEvent',
+                             ['name', 'keys', 'func', 'args', 'kwargs'])
+
+
+class _GlobalKeyEvents(object):
+    def __init__(self):
+        self.events = {}
+        self.add('shutdown', prefs.general['shutdownKeys'], core.quit)
+
+    def __repr__(self):
+        return '%s' % self.events
+
+    def add(self, name, keys, func, *args, **kwargs):
+        """
+        Add a global event key.
+
+        :Parameters:
+
+        name : string
+            The name of the event to create. If an event with the specified
+            named already it exists, it will be replaced.
+
+        keys : string or iterable
+            A single key or list of keys.
+
+        func : function
+            The function to invoke once the specified keys were pressed.
+
+        args, kwargs
+            All additional a rguments and keyword arguments are passed to
+            the specified function.
+
+        """
+        if isinstance(keys, (str, unicode)):
+            keys = [keys]
+
+        for key in keys:
+            self.events[key] = _GlobalKeyEvent(name, keys, func, args, kwargs)
+
+    def remove(self, keys=None):
+        """
+        Remove global event keys.
+
+        keys : string, iterable, or None
+            A single key or list of keys. If None, remove all event keys.
```

Comment:
This is not intuitive. "Remove None" sounds like removing nothing. Either we should have an `globalKeys.clear()` (as in events.clearEvents()) or `globalKeys.remove('all')`